AdobeStockImage/Api/Data/KeywordInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\AdobeStockImage\Api\Data;
+
+interface KeywordInterface
```

Comment:
It would be great to have methods for retrieving/setting id and adobe_id fields in the KeywordInterface

Reply:
@sivaschenko, the adobe_stock_keyword table has only the id and the keyword fields: http://prntscr.com/nu7dgj

Reply:
@igor-imaginemage you are correct! Than only getter/setter for `id` is needed